### PR TITLE
Nested code owners did not work, moving Rohan to the global config file

### PR DESCRIPTION
**Pull request by nadiam84**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1,9 @@
-* @JedWatson @nadiam84 @treshugart
\ No newline at end of file
+## global code owners
+* @JedWatson @nadiam84 @treshugart
+
+## SPA code owners
+/guides/react/app/routing/
+* @rohan-deshpande @treshugart
```
Comment (treshugart) [inline CODEOWNERS]: I *think* my global ownership makes this redundant. Let's merge and test!

Comment (treshugart) [reply CODEOWNERS]: Ahh wait, looks like we need the path next to the names. I believe this should look like:

```
## SPA code owners
/guides/react/app/routing @rohan-deshpande
```

Comment (nadiam84) [reply CODEOWNERS]: ooops :coneofshame:

Comment (nadiam84) [reply CODEOWNERS]: > I _think_ my global ownership makes this redundant. Let's merge and test!

In the docs: "# Order is important; the last matching pattern takes the most
# precedence. When someone opens a pull request that only
# modifies JS files, only @js-owner and not the global"

So basically global owners are slightly useless, we would need to add them everywhere :(

But let's test :D

Comment (treshugart) [reply CODEOWNERS]: Yeah let's test. I think that means that our global should go last. Sounds like that'd satisfy our use case?

**Pull request merged**